src/Views/Bookmark.elm

```diff
@@ -0,0 +1,268 @@
+module Views.Bookmark exposing (ActiveTab(..), Scope(..), view)
+
+import Data.Bookmark as Bookmark exposing (Bookmark)
+import Data.Impact as Impact
+import Data.Session exposing (Session)
+import Data.Unit as Unit
+import Html exposing (..)
+import Html.Attributes exposing (..)
+import Html.Events exposing (..)
+import Page.Textile.Simulator.ViewMode as ViewMode exposing (ViewMode)
+import Route
+import Time
+import Views.Icon as Icon
+
+
+type alias ManagerConfig msg =
+    { session : Session
+    , activeTab : ActiveTab
+    , bookmarkName : String
+    , impact : Impact.Definition
+    , funit : Unit.Functional
+    , viewMode : ViewMode
+    , scope : Scope
+
+    -- Messages
+    , copyToClipBoard : String -> msg
+    , compare : msg
+    , delete : Bookmark -> msg
+    , save : msg
+    , update : String -> msg
+    , switchTab : ActiveTab -> msg
+    }
+
+
+type Scope
+    = Food
+    | Textile
+
+
+type ActiveTab
+    = SaveTab
+    | ShareTab
+
+
+view : ManagerConfig msg -> Html msg
+view ({ activeTab, switchTab } as config) =
+    div [ class "card shadow-sm" ]
+        [ div [ class "card-header" ]
+            [ [ ( SaveTab, "Sauvegarder" ), ( ShareTab, "Partager" ) ]
+                |> List.map
+                    (\( msg, label ) ->
```

Comment:
how about renaming `msg` to `tab`? I'm not sure i understand the meaning of `msg` here

Reply:
`SaveTab ` and `SaveTab` are member of the `Msg` type, that's why.

Reply:
so they are not the variants of the `ActiveTab` custom type above?

Reply:
Oooh my mistake, I was confused indeed, fixing this right now